csharp/sources/Valkey.Glide/Internals/Request.SortedSetCommands.cs

```diff
@@ -64,4 +64,31 @@ public static Cmd<long, long> SortedSetAddAsync(ValkeyKey key, SortedSetEntry[]
 
         return Simple<long>(RequestType.ZAdd, [.. args]);
     }
+
+    public static Cmd<long, bool> SortedSetRemoveAsync(ValkeyKey key, ValkeyValue member, CommandFlags flags = CommandFlags.None)
+    {
+        if (flags != CommandFlags.None)
+        {
+            throw new NotImplementedException("Command flags are not supported by GLIDE");
+        }
```

Comment:
Can probably replace with
```suggestion
		Utils.Requires<NotImplementedException>(flags == CommandFlags.None, "Command flags are not supported by GLIDE");
```

Reply:
Updated.